validation/jinja2/validation/_segment_card.html

```diff
@@ -33,7 +33,13 @@
                 {% if auth %}
                     <button type="button" class="button button--fail button--flag right" data-toggle="modal" data-target="#{{ phrase.id }}-modal" data-cy="flag-button">Something's wrong with this entry</button>
                 {% endif %}
-                <p class="card__title--cree" data-cy="transcription">{{ phrase.transcription }}</p>
+                <p class="card__title--cree" data-cy="transcription">
+                    {% if request.COOKIES['macron'] == 'on' %}
+                        {{ phrase.transcription }}
+                    {% else %}
+                        {{ phrase.transcription | replace("ê", "e") }}
```

Comment:
Pedantic note: the diacritic on the ê [is a circumflex](https://en.wikipedia.org/wiki/Circumflex#Length:~:text=In%20western%20Cree%2C); a macron is [a straight line on top](https://en.wikipedia.org/wiki/Macron_(diacritic)#cite_ref-15:~:text=In%20western%20Cree), like this: ē.

In ALTLab-internal usage, I prefer using the precise names, but I understand that normal folks call both "macrons" — circumflex is such a weird word!

Reply:
This is valid. I was lowkey waiting to see if someone would comment on the wording. I can change it to circumflex and see if we get any user feedback about the wording.